server/bootserver.go

```diff
@@ -147,7 +137,11 @@ func handleIPXE(w http.ResponseWriter, r *http.Request, k8sClient client.Client,
 			SquashfsURL:   config.Spec.SquashfsURL,
 			IPXEServerURL: ipxeServiceURL,
 		})
+		return
 	}
+
+	log.Info("No UUID is specified, delivering default script")
```

Comment:
I would remove this as well, unless there is a usecase for it. 
I believe, FeDHCP/DNSMASQ should always look for /ipxe/uuid even while discovering/onboading the server.

Reply:
good point, I agree to drop this